dev/docker-compose.yml

```diff
@@ -40,6 +36,8 @@ services:
     - pg:pg
     - ldap-server
     - oidc-keycloak:keycloak
+    command:
```

Comment:
Can we use volume instead?

Reply:
Could you please rephrase that? I'm not sure I understand what you mean.